test/unit/alphabet/semi_alphabet_constexpr_test_template.hpp

```diff
@@ -0,0 +1,156 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
+
+#include <seqan3/alphabet/concept.hpp>
+
+using namespace seqan3;
+
+template <typename T>
+using semi_alphabet_constexpr = ::testing::Test;
+
+TYPED_TEST_CASE_P(semi_alphabet_constexpr);
+
+TYPED_TEST_P(semi_alphabet_constexpr, concept_check)
+{
+    EXPECT_TRUE(detail::constexpr_semialphabet<TypeParam   >);
```

Comment:
So I can now read it better because the bracket closes with the one below 🤦‍♂ (sorry this irony had to be 😁 )
However, I would be in strong favor of removing these empty spaces at the end. 💅

Reply:
1. This is copy and pasted from hannes tests. It was just moved into a different file.
2. Considering that you and marcel were the ones complaining about too much irony in review comments, you seem to use a lot of irony in your last 5 reviews of my code. :wink: